target/linux/realtek/dts/rtl930x.dtsi

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
This patch is about adding a timer, so I don't see how adding the full DTSI belongs here.